cg/services/deliver_files/deliver_files_service/deliver_files_service.py

```diff
@@ -89,6 +89,7 @@ def deliver_files_for_ticket(
             )
 
     def _start_rsync_job(self, case: Case, dry_run: bool, folders_to_deliver: set[Path]) -> int:
+        LOG.debug(f"[RSYNC] Starting rsync job for case {case.internal_id}")
```

Comment:
Do you like this logging? @Vince-janv I add the service at the beginning of the error message